scripts/requirements-doc.txt

```diff
@@ -10,3 +10,4 @@ azure-storage-blob
 sphinx_markdown_tables
 markdown<3.3.5 # Workaround for ryanfox/sphinx-markdown-tables#34
 mistune<2.0 # Workaround for https://github.com/CrossNox/m2r2/issues/40
+breathe!=4.33 # Workaround for https://github.com/michaeljones/breathe/issues/803
```

Comment:
this may need to specify 4.33.0 instead of just the major.minor semantic. fix is available in 4.33.1